test/java/org/opendatakit/briefcase/ui/export/components/ConfigurationPanelPageObject.java

```diff
@@ -67,23 +67,19 @@
     }
   }
 
-  private ExportConfiguration currentConf;
```

Comment:
new/revisit

All the *PageObject classes have to be thoroughly reviewed.

UI component tests should be revisited to see how well they're helping us prevent bugs and see how we can add more tooling to have more of them.